app/components/create_log_actions_component.html.erb

```diff
@@ -6,5 +6,8 @@
     <% if upload_button_href.present? && !user.support? %>
       <%= govuk_button_link_to upload_button_copy, upload_button_href, secondary: true %>
     <% end %>
+    <% if view_uploads_button_href.present? && user.support? %>
```

Comment:
What would be scenario where `view_uploads_button_href` doesn't exist?

Reply:
None really, this component is only used on logs/index and the log_type is set from the controller. Which it should always be one of lettings or sales. I think I can refactor this component and remove the .present? checks